### PR TITLE
singly linked list

### PR DESCRIPTION
**Pull request by simmi01**

### CODE REVIEW

**Review by Harshada-github — CHANGES_REQUESTED** [review]
Remove extra blank lines.

File: Simran/Sim_C_Program/singly_linked_list.c [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+void insert_list_in_end();
+void delete_list_from_beg();
+void delete_list_from_end();
+void display_list();
+void search_list();
+
+
+
+int 
+main() {
```
Comment (shankar7791) [inline Simran/Sim_C_Program/singly_linked_list.c]: this curlt brace should be on new line
only control statements corly brace is on same line

File: Simran/Sim_C_Program/singly_linked_list.c [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+	while(1) {
+	
+	
+		printf("\n ------------ SINGLY LINKED LIST OPERATION ------------");
+		printf("\n 1. INSERT \n 2. DELETE \n 3. SEARCH \n 4. DISPLAY \n 5. EXIT ");
+		printf("\n Eter your choice : ");
+		scanf("%d",&choice);
+		
+		switch(choice)
+		{
```
Comment (shankar7791) [inline Simran/Sim_C_Program/singly_linked_list.c]: this curly brace is same line as switch statment

File: Simran/Sim_C_Program/singly_linked_list.c [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+			
+			case 1 :
+				printf("\n ~~~ Select Insertion Position ~~~ ");
+				printf("\n 1. Inserting Node in the Beginning. \n 2. Inserting Node in the End. ");
+				scanf("%d" , &choice);
```
Comment (shankar7791) [inline Simran/Sim_C_Program/singly_linked_list.c]: space is after " , " not before " , " eg : scanf("%d", &num)

File: Simran/Sim_C_Program/singly_linked_list.c [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+		}
+		
+		if(flag == 1)
+			printf("\n Item is not present in the list !");
+	}
+}
+	
+
+	
+
```
Comment (shankar7791) [inline Simran/Sim_C_Program/singly_linked_list.c:171]: remove extra line spaces

File: Simran/Sim_C_Program/singly_linked_list.c [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+	
+	else {
+	
+		temp = head;
+		head = head -> next;
+		printf("\n Deleted element is %d" , temp -> data);
+		free(temp);
+	}
+}
+
```
Comment (shankar7791) [inline Simran/Sim_C_Program/singly_linked_list.c:115]: remove extra line spaces in all programe

**Review by shankar7791 — CHANGES_REQUESTED** [review]
change the coding style

**Pull request merged**